components/brave_wallet/browser/brave_wallet_prefs.cc

```diff
@@ -54,6 +54,11 @@ constexpr char kBraveWalletUserAssetsAddIsSpamMigrated[] =
 constexpr char kBraveWalletUserAssetsAddIsERC1155Migrated[] =
     "brave.wallet.user.assets.add_is_erc1155_migrated";
 
+constexpr char kBraveIpfsFeatureMigrated[] = "brave.ipfs";
+constexpr char kBraveIpfsCompanionMigrated[] = "brave.ipfs_companion_enabled";
+constexpr char kPinnedNFTAssetsMigrated[] = "brave.wallet.user_pin_data";
+constexpr char kAutoPinEnabledMigrated[] = "brave.wallet.auto_pin_enabled";
+
```

Comment:
Let's clear there only wallet prefs.

Reply:
Also please add a comment with deprecation date (06/2024?) so we can remove them completely from code after a year